docs/Other SDL Features/Setting the Navigation Destination/index.md

```diff
@@ -48,15 +48,32 @@ sdlManager.getSystemCapabilityManager().getCapability(SystemCapabilityType.NAVIG
 	@Override
 	public void onCapabilityRetrieved(Object capability) {
 		NavigationCapability navCapability = (NavigationCapability) capability;
-		Boolean isNavigationSupported = navCapability.getSendLocationEnabled();
+		Boolean isNavigationSupported = navCapability != null && navCapability.getWayPointsEnabled();
```

Comment:
This should be `getSendLocationEnabled()` or something like that, right?

Reply:
Yes. Fixed